assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -240,9 +240,47 @@ function APIProvider({ children }) {
     return apiFetch({ path });
   }, [statuses]);
 
-  const getAllUsers = useCallback(() => {
-    return apiFetch({ path: users });
-  }, [users]);
+  const getUsersByPage = useCallback(
+    (pagingNum = 1) => {
+      let apiPath = users;
+      const perPage = 10;
```

Comment:
I think that still should remain 100. On my local dev, I have 564 users and this ended up in 56 requests. Which feels a like crazy. It also took a lot longer to resolve.

Reply:
Yes, it should be 100! And I've got 108 users and got successfully.